README.md

```diff
@@ -1 +1,170 @@
+  <a href="#-installation">Installation</a> •  
+  <a href="#-️getting-started">Getting Started</a> 
+</p>  
+</p>
+
+<h2 align="center">What is it</h3>  
+  
+**Spello** is a spellcorrection model using power of two model Symspell and Phoneme in backend to get the best possible spelling suggestion for misspelled words in a text. Phoneme Model uses <a href="https://en.wikipedia.org/wiki/Soundex">Soundex</a> algo in background and is responsible to suggest corrections for sound related mistakes in word. Also, we are using a modifield version of<a href="https://github.com/wolfgarbe/SymSpell"> Symspell</a> model to get spell suggestions based on edit-distances. <br>
+Currently, this module is available for **English(en)** and 10 other indian languages which are **Hindi(hi)**,  **Marathi(mr)**,  **Bengali(bn)**,  **Punjabi(pa)**, **Gujarati(gu)**, **Oriya(or)**, **Tamil(ta)**, **Telegu(tl)**, **Kannada(kn)**, **Malyalam(ml)**.  
+  
+<h2 align="center">💾 Installation</h2>  
+<p align="right"><a href="#what-is-it"><sup>▴ Back to top</sup></a></p>
+Install the spello via `pip`
+
+```bash  
+$ pip install spello
+```  
+
+<h2 align="center">⚡ ️Getting Started</h2> 
+<p align="right"><a href="#what-is-it"><sup>▴ Back to top</sup></a></p>
+  
+#### 1. **Model Initialisation**
+Initialise the model for one of the suppored languages. 
+```python  
+>>> from spello.model import SpellCorrectionModel  
+>>> sp = SpellCorrectionModel(language='en')  
+```  
+
+#### 2. Model Training
+Here, you can provide two types of training data to the model
+- List of text or sentences.
+- Dict having word and their corresponding count.
+
+**Training providing list of sentences**
+```python 
+>>> sp.train(['my name is aman', 'this is a text corpus'])
+```
+**Training providing words counter**
+```python 
+>>> sp.train({'my': 2, 'name': 1, 'aman': 1, 'text': 10, 'corpus': 5})
+```
+> List of text is a recommended type for training data as here model also tries to learn context in which words are appearing, which further help to find best possible suggestion in case more than one suggestions are suggested by symspell or phoneme model
+
+#### 3. Model Prediction
+```python  
+>>> sp.spell_correct('my naem is naman')  
+{'original_text': 'my naem is naman',
+ 'spell_corrected_text': 'my name is aman',
+ 'correction_dict': {'naem': 'name', 'naman': 'aman'}
+}
+```  
+
+#### 4. Save Model
+Call the save method to save the trained model at given model dir 
+```python  
+>>> sp.save(model_save_dir='/home/ubuntu/')
+'/home/ubuntu/model.pkl' # saved model path
+```  
+
+#### 5. Load Model 
+Load the trained model from saved path, First initialise the model and call the load method
+```python  
+>>> from spello.model import SpellCorrectionModel
+>>> sp = SpellCorrectionModel(language='en')
+>>> sp.load('/home/ubuntu/model.pkl')
+```  
+
+#### 6. Customize Configuration of Model
+Here, you are also provided to customize various configuration of the model like 
```

Comment:
"You can choose to customize various following configurations of the model"

Reply:
Done